op-deployer/pkg/deployer/pipeline/alt_da.go

```diff
@@ -27,11 +27,16 @@ func DeployAltDA(env *Env, intent *state.Intent, st *state.State, chainID common
 		return nil
 	}
 
+	chainIndex, err := findChainIndex(st.Chains, chainID)
```

Comment:
[nit] There's already a util for this - `st.StateFor(chainID)`

Reply:
https://github.com/ethereum-optimism/optimism/pull/12904/commits/ccd954ebc5c4d06da92537994c6abee9ad08666a